alf/algorithms/algorithm.py

```diff
@@ -161,6 +160,10 @@ def __init__(self,
         if optimizer:
             self._optimizers.append(optimizer)
         self._is_on_policy = is_on_policy
+        if config:
+            self._rl_train_every_update_steps = config.rl_train_every_update_steps
+            self._rl_train_after_update_steps = config.rl_train_after_update_steps
+        self._count = 0
```

Comment:
Should be a better name. Iteration? Update? what is its relationship with global counter?

Reply:
Changed to update_count. Originally thought about using global counter, but this counter seems to be impacted by a train_from_replay_buffer flag (although we set it to be True typically).

Reply:
Removed _count.